test/basics.jl

```diff
@@ -340,4 +340,21 @@ df = CSV.read(IOBuffer("a,b,c\n1,2,3\n\n"), ignoreemptylines=true)
 df = CSV.read(IOBuffer("zip\n11111-1111\n"), dateformat = "y-m-dTH:M:S.s")
 @test size(df) == (1, 1)
 
+# Supporting commands across multiple platforms cribbed from julia/test/spawn.jl
+catcmd = `cat`
+havebb = false
+if Sys.iswindows()
+    busybox = download("https://frippery.org/files/busybox/busybox.exe", joinpath(tempdir(), "busybox.exe"))
```

Comment:
According to [this SO question](https://superuser.com/questions/434870/what-is-the-windows-equivalent-of-the-unix-command-cat), it seems like we could just call `type $file`?

Reply:
I thought so too.  But commit 2bde38b doesn't pass regression...